sysl2/sysl/cmd_runner.go

```diff
@@ -51,8 +61,8 @@ func (r *cmdRunner) Configure(app *kingpin.Application) error {
 	r.commands = map[string]Command{}
 
 	app.Flag("root",
-		"sysl root directory for input model file (default: .)").
-		Default(".").StringVar(&r.Root)
+		"sysl root directory for input model file. If root is not found, the module directory becomes the root").
```

Comment:
This docstring implies that, in the absence of a root, `import /foo` will look for a file in the same directory as the importing file. It should fail.